pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.black]
+line-length=119
+target-version=["py38"]
+
+[tool.isort]
+atomic=true
+line_length = 119
+lines_after_imports=2
+lines_between_types=1
+use_parentheses = true
+balanced_wrapping = true
+include_trailing_comma = true
+multi_line_output = 3
+known_third_party=[
+    "pytest",
+]
+known_first_party=["convert2rhel"]
+
+[tool.coverage.run]
+branch = false
+source = ["convert2rhel"]
+omit = ["convert2rhel/unit_tests"]
+
+[tool.coverage.paths]
+source = ["convert2rhel/"]
+
+[tool.coverage.report]
+fail_under = 87
+skip_covered = true
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.2.1"
+addopts = "-vv -s --tb=native"
+testpaths = "convert2rhel/unit_tests"
+log_cli = true
+log_cli_level = "DEBUG"
+log_cli_format = "| %(asctime)s | %(name)s | %(levelname)s | %(filename)s | %(message)s"
+
+[tool.interrogate]
+ignore-init-method = true
+ignore-init-module = true
+ignore-magic = false
+ignore-semiprivate = false
+ignore-private = false
+ignore-property-decorators = false
+ignore-module = true
+fail-under = 100
+exclude = ["setup.py", "docs", "build", "tests/load_testing"]
+ignore-regex = [
+    "^get$",
+    "^perform.*$",
+    "^mock_.*",
+    ".*BaseClass.*",
+    "^get.*$",
+    "^factory$",
+    "^_.+$",
+]
+verbose = 2
+quiet = false
+whitelist-regex = []
+color = true
+
+[tool.poetry]
```

Comment:
What is poetry? How can I verify all its setup is correct?

Reply:
that was added accidentally - now no poetry here